packages/react/src/React.ml

```diff
@@ -339,8 +339,8 @@ module JSX = struct
     | Inline of string
 
   type prop =
-    | Bool of (string * bool)
-    | String of (string * string)
+    | Bool of ((string * string) * bool)
```

Comment:
I would use `string * string * bool` instead, wdyt?

Reply:
It's better to keep a single tuple for perf, I haven't tested and just guessing but contextualisation here doesn't make much difference